test/testmachinery/shoots/logging/seed_logging_stack.go

```diff
@@ -399,7 +633,7 @@ func prepareFluentBitServiceAccount(serviceAccount *corev1.ServiceAccount) *core
 	return serviceAccount
 }
 
-func getFluentBitDaemonSet(ctx context.Context, k8sSeedClient kubernetes.Interface) (*appsv1.DaemonSet, error) {
+func getFluentBitDeamonSet(ctx context.Context, k8sSeedClient kubernetes.Interface) (*appsv1.DaemonSet, error) {
```

Comment:
It is DaemonSet, not DeamonSet - see https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/

```suggestion
func getFluentBitDeamonSet(ctx context.Context, k8sSeedClient kubernetes.Interface) (*appsv1.DaemonSet, error) {
```

Reply:
type is fixed